src/style.css

```
@@ -1,3 +1,92 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: gray;
+  text-transform: uppercase;
+
+  --blue: rgba(0, 172, 220, 1);
+  --white: rgba(255, 255, 255, 1);
 }
+
+.header {
+  margin: 0 auto;
+  background-color: var(--white);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__image {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+display: flex;
+margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  height: 60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto 10px;
+
+}
+
+.first {
+  margin-left: 0;
+}
+
+.last {
+  margin-right: 0;
+}
+
+
+.nav__link {
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  color: #000;
+  height: 60px;
+}
+
+a:hover {
+  color: var(--blue);
+  ;
+}
```

it's better not to use tag names for styling (later in project you can have more link elements)
```suggestion
.nav__link:hover {
  color: var(--blue);
}
```